mito-ai/src/Extensions/AiChat/ChatMessage/ChatMessage.tsx

```diff
@@ -74,7 +127,23 @@ const ChatMessage: React.FC<IChatMessageProps> = ({
                     }
                 } else {
                     return (
-                        <p>{mitoAIConnectionError && <span style={{marginRight: '4px'}}><ErrorIcon /></span>}{messagePart}</p>
+                        <div style={{ position: 'relative' }}>
+                            <p onDoubleClick={() => setIsEditing(true)}>
+                                {mitoAIConnectionError && <span style={{ marginRight: '4px' }}><ErrorIcon /></span>}
+                                {messagePart}
+                            </p>
+                            {message.role === 'user' && (
+                                <div style={{ display: 'flex', justifyContent: 'flex-end', marginTop: '4px' }}>
+                                    <button
```

Comment:
Let's add a title tag here too so when the user hovers over the icon it says "Edit message" or something like that.